### ISSUE TITLE
[apollo-server-restify]: restify is outdated

### ISSUE THREAD

**Issue by liyucun** [apollographql/apollo-server#472]
The current restify version is outdated. Since its type def changed a little bit, the current demo example doesn't work. It will throw type errors for `  server.post("/graphql", graphqlRestify(graphQLOptions))`:

```
Argument of type 'RestifyHandler' is not assignable to parameter of type 'RequestHandlerType'. 
```

I would love to do a quick update PR on this, and please let me know if you guys currently work on it.

**Comment by martijnwalraven** [comment]
Thanks for letting us know. I don't think anyone is currently working on this, so a PR is definitely welcome!

**Comment by martijnwalraven** [comment]
This has just been published in Apollo Server 1.1.0.
